src/table/TableBodyWrapper.jsx

```diff
@@ -126,8 +127,12 @@ TableBodyWrapper.propTypes = {
   selectionsAPI: PropTypes.object.isRequired,
   layout: PropTypes.object.isRequired,
   theme: PropTypes.object.isRequired,
-  focusedCellCoord: PropTypes.object.isRequired,
+  focusedCellCoord: PropTypes.arrayOf(PropTypes.number),
+  handleFocusedCellCordsUpd: PropTypes.func.isRequired,
   setShouldRefocus: PropTypes.func.isRequired,
 };
 
-export default React.memo(TableBodyWrapper);
+export default memo(
+  TableBodyWrapper,
+  (prevProps, nextProps) => prevProps.focusedCellCoord.toString() !== nextProps.focusedCellCoord.toString()
```

Comment:
I'd say this needs a comment. Unless you know about this fix, it's hard to understand when it updates or not. same thing in headWrapper

Reply:
That was why i used `if ... else` before in order to clear it up a bit in. By default the logic behind comparing `prevProps` and `nextProps` always confusing, unless you make it clear that for example i want to return true in these cases and false in other ones. 

My suggestion here could be the same `if...else` because it's a bit clear and simple to understand in this case.

Reply:
That is just bad syntax IMO. Not that hard to understand the condition itself, but why it should update when focusCellCoord stays the same, and not update when it changes. Kind of counter-intuitive. Usually you update when something else changes. That part needs an explanation I think

Reply:
✅ Done, I added this: 
```
we need to check if `focusedCellCoord` updated, then no need to re-render this component
because it causes unnecessary extra re-render to this component when we move the focus to another cell and we want to avoid that
```